oss-internship-2020/sapi_lodepng/README.md

```diff
@@ -0,0 +1,42 @@
+# LodePNG Sandboxed API
+
+Sandboxed version of the [LodePNG](https://github.com/lvandeve/lodepng) library, using [Sandboxed API](https://github.com/google/sandboxed-api)
+
+## Details
+
+With Sandboxed API, many of the library's functions can be sandboxed. However, they need the `extern "C"` keyword defined so that name mangling does not happen, which is why a fork of the **LodePNG** library is used. The only differences are found in the header file. An alternative to this is to define another library that wraps every needed function, specifying the required keyword.
```

Comment:
We're not really using a fork but patch the code (as you correctly explain in the next paragraph), maybe reword this?